README.md

```
@@ -35,11 +35,33 @@ _I'd love for you to make the most of this project - it's all about learning, he
+
+	# verifies the right Node.js version is in the environment
+	node -v # should print `v20.18.0`
+
+	# verifies the right npm version is in the environment
+	npm -v # should print `10.8.2`
    npm i
    ```
 
 4. **Set up your MongoDB Database**
-
+	sudo apt-get install gnupg curl
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)